{"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"paths":["\/documentation\/core\/keychainerror\/unhandlederror(message:)"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Core","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Core\/KeychainError"]]},"sections":[],"abstract":[{"text":"An unhandled error occurred perform the keychain operation.","type":"text"}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unhandledError","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"externalID":"s:4Core13KeychainErrorO09unhandledC0yACSS_tcACmF","role":"symbol","title":"KeychainError.unhandledError(message:)","symbolKind":"case","roleHeading":"Case","modules":[{"name":"Core"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unhandledError"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"platforms":["iOS"]}]}],"identifier":{"url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Core\/KeychainError\/unhandledError(message:)","interfaceLanguage":"swift"},"references":{"doc://com.ibm.verify.ios.sdk/documentation/Core":{"kind":"symbol","role":"collection","url":"\/documentation\/core","title":"Core","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Core","type":"topic","abstract":[]},"doc://com.ibm.verify.ios.sdk/documentation/Core/KeychainError":{"type":"topic","url":"\/documentation\/core\/keychainerror","abstract":[{"type":"text","text":"An error that occurs during keychain operations."}],"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KeychainError"}],"kind":"symbol","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Core\/KeychainError","title":"KeychainError","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"KeychainError"}]},"doc://com.ibm.verify.ios.sdk/documentation/Core/KeychainError/unhandledError(message:)":{"title":"KeychainError.unhandledError(message:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"unhandledError","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"An unhandled error occurred perform the keychain operation."}],"role":"symbol","type":"topic","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Core\/KeychainError\/unhandledError(message:)","kind":"symbol","url":"\/documentation\/core\/keychainerror\/unhandlederror(message:)"}}}